design/plugin.md

```diff
@@ -95,6 +95,8 @@ While BIN plugins are not deprecated, they do not support the same set of featur
 
 As its name suggests, this command removes plugins (their name being given as arguments) locally by removing their directory from the filesystem.
 
+As the [core CLI is bundled](corecli.md) within the DC/OS CLI, trying to remove the `dcos-core-cli` plugin would trigger an error.
```

Comment:
s/trying to remove the `dcos-core-cli` plugin would trigger an error./removing the `dcos-core-cli` plugin triggers an error.